### ISSUE TITLE
Installation problem with djangorestframework

### ISSUE THREAD

**Issue by Lapin-Blanc** [ionyse/ionyweb#8]
Hello, I just tried to install, but I have issues with djangorestframework
This one is installed correctly as a dependency,  but can't be imported.
I noticed that it now has to be imported as "rest_framework" instead of "djangorestframework"
So I symlinked the rest_framework to djangorestframework...
Again, it seems that the package has evolved too much, as I now have new import errors : 

```
from djangorestframework.response import ErrorResponse :
ImportError: cannot import name ErrorResponse
```

**Comment by Lapin-Blanc** [comment]
Solved when editing setup.py to force djangorestframework==0.4.0

**Comment by Natim** [comment]
Force the 0.4 version.

**Comment by Lapin-Blanc** [comment]
This is what I just did... Works fine now ;-)
Btw, I'm stuck on the template part, can't find the "modulo" mentioned in sources, should I build a complete template/layout to try things out ?

**Comment by Natim** [comment]
no have a look at http://github.com/ionyse/fimu

**Comment by Natim** [comment]
Fixed in 0.2
